jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -219,6 +220,8 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 		try {
 			FileManager.initialise(filePath);
 		} catch (IOException e1) {
+			log.error("Error occurred in job azure-data-collector-job. Failed to create file in S3 in the given path");
```

Comment:
not sure that this log is useful

Reply:
gave reply above.

Reply:
it will print our message first followed by error stack trace.